### PR TITLE
Change groupid to com.squareup.okhttp3.

### PR DESCRIPTION
**Pull request by swankjesse**

### CODE REVIEW

File: mockwebserver/pom.xml [hunk 1]
```diff
@@ -14,18 +14,18 @@
 
   <dependencies>
     <dependency>
-      <groupId>com.squareup.okhttp</groupId>
+      <groupId>com.squareup.okhttp3</groupId>
```
Comment (JakeWharton) [inline mockwebserver/pom.xml]: If you care to, all these dependencies on sibling artifacts can use `${project.groupId}`.

Comment (swankjesse) [inline mockwebserver/pom.xml]: Seems like a win. Will save me a few minutes in 2025 when we do okhttp4 !

**Comment by JakeWharton** [comment]
LGTM after that change

File: mockwebserver/pom.xml (hunk 1 above)
Comment (swankjesse) [inline mockwebserver/pom.xml]: Done!

**Pull request merged**